core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -412,49 +413,58 @@ class Router(formatOps: FormatOps) {
 
       // Parameter opening for one parameter group. This format works
       // on a group-by-group basis.
-      case FormatToken(open @ LeftParen(), right, between)
-          if style.verticalMultiline && isDefnSite(leftOwner) =>
+      case FormatToken(open @ LeftParen(), _, between)
+          if style.verticalMultilineAtDefinitionSite && isDefnSite(leftOwner) =>
         val close = matchingParentheses(hash(open))
         val indentParam = Num(4)
         val indentSep = Num(2)
-        if (isTuple(leftOwner)) {
-          Seq(
-            Split(NoSplit, 0).withPolicy(
-              SingleLineBlock(close, disallowInlineComments = false))
-          )
-        } else {
-          val oneLinePerArgPolicy =
-            OneArgOneLineSplit(
-              open,
-              noTrailingCommas = style.poorMansTrailingCommasInConfigStyle)
-
-          // A `paramSep` is a `)(`, aka the seperator between two parameter
-          // groups. Prepend a newline to these.
-          val noNewLineOnParamSep: PartialFunction[Decision, Decision] = {
-            case Decision(t @ FormatToken(_, `close`, _), _) =>
-              Decision(t,
-                Seq(Split(Newline, 0)
-                  .withIndent(indentSep, close, Right)
-                )
-              )
+
+        @tailrec
+        def loop(token: Token): Option[Token] =
```

Comment:
Can we move this to FormatOps with a clearer method name? Suggestion: lastClosingParam with return type Token.RightParen, no option since it can't be that by construction of the map. Exceptions are caught and dealt with higher up.